src/app/components/PageLayout/Header.tsx

```diff
@@ -38,27 +36,20 @@ export const Header: FC = () => {
         boxShadow: '0px 4px 6px -1px rgba(0, 0, 0, 0.10), 0px 2px 4px -1px rgba(0, 0, 0, 0.06)',
       }}
     >
-      <Box sx={{ px: '15px' }}>
-        <Grid
-          container
-          sx={{
-            px: isMobile ? 0 : '4%',
-            pt: isMobile ? 4 : '15px',
-            pb: 4,
-          }}
-        >
-          <Grid lg={3} xs={4} sx={{ display: 'flex', alignItems: 'center' }}>
+      <div className="px-4">
+        <div className="flex justify-between pt-3 pb-4 px-0 md:px-[4%]">
+          <div className="w-1/3 lg:w-1/4 flex items-center">
```

Comment:
just curious why we don't migrate layout to new grid system `grid grid-cols-12 / col-span-4 lg:col-span-3` etc ?